Update Tizen SDK (#1602)
Samsung/tizen-docs: docs/platform/what-is-tizen/overview.md

## Productivity

> To facilitate developing Tizen applications, [Tizen Studio](/application/tizen-studio/) is provided as an integrated development environment (IDE). It helps you to develop, build, debug, profile, and emulate Tizen applications. Tizen Studio is equipped with useful developer tools, including emulator, and Dynamic Analyzer for profiling. In the integrated environment, you can enjoy comfortable and productive development of Tizen Native and Web applications. Also, we provide two extensions for Visual Studio (IDE) and Visual Studio Code. In "[Visual Studio Tools for Tizen](application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not Tizen Web and Native applications but also Tizen .NET applications easily and rapidly, and use Tizen Studio'tools ,too.

Reviewer: ```suggestion
To facilitate developing Tizen applications, [Tizen Studio](/application/tizen-studio/) is provided as an integrated development environment (IDE). It helps you to develop, build, debug, profile, and emulate Tizen applications. Tizen Studio is equipped with useful developer tools, including Emulator, Certificate Manager, Device Manager, and Dynamic Analyzer for profiling. In the integrated environment, you can enjoy comfortable and productive development of Tizen Native and Web applications. Also, we provide two extensions for Visual Studio and Visual Studio Code. In "[Visual Studio Tools for Tizen](application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not only Tizen Web and Native applications but also Tizen .NET applications easily and rapidly, and use Common Tools.
```

Reviewer 2: Hello.
I have a question related with this comment.

1. To my knowledge, "Common Tool" is used on menu in PR 1600, to gather the tools that can be used in Tizen Studio, Visual Studio and Visual Studio Code. (Mostly from Baseline SDK)
   It means, "Common Tool" is a name used on menu for convenience's sake, rather than official tool's name

   So I'm worried with "and use Common Tools.", as it can give confusion that there are tools really named "Common Tools"

2. Emulator, Certificate Manager, Device Manager are under "Common Tools" menu.
   So if we write those tools separately in Tizen Studio description and  Common tools, to me it sounds that "Emulator, Certificate Manager..." and "Common Tools" are different tools.

So how about below?

```suggestion
To facilitate developing Tizen applications, [Tizen Studio](/application/tizen-studio/) is provided as an integrated development environment (IDE). It helps you to develop, build, debug, profile, and emulate Tizen applications. Tizen Studio is equipped with useful developer tools, including Emulator, Certificate Manager, Device Manager, and Dynamic Analyzer for profiling. In the integrated environment, you can enjoy comfortable and productive development of Tizen Native and Web applications. Also, we provide two extensions for Visual Studio and Visual Studio Code. In "[Visual Studio Tools for Tizen](application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not only Tizen Web and Native applications but also Tizen .NET applications easily and rapidly with Tizen Extension.
```

Reviewer 2: On chat, @sangwook96 suggested

```suggestion
To facilitate developing Tizen applications, [Tizen Studio](/application/tizen-studio/) is provided as an integrated development environment (IDE). It helps you to develop, build, debug, profile, and emulate Tizen applications. Tizen Studio is equipped with useful developer tools, including Emulator, Certificate Manager, Device Manager, and Dynamic Analyzer for profiling. In the integrated environment, you can enjoy comfortable and productive development of Tizen Native and Web applications. Also, we provide two extensions for Visual Studio and Visual Studio Code. In "[Visual Studio Tools for Tizen](application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not only Tizen Web and Native applications but also Tizen .NET applications easily and rapidly with tools which Tizen Studio provide.
```

Reviewer 2: @asm-foysal I want to know if you can suggest us a sentence.

"Emulator, Certificate Manager, Device Manager, and Dynamic Analyzer" are separative tools with Tizen Studio. (not supported by Tizen Studio)
That is why we can use those tools with VS and VSCode.

In this case, do you have any idea to change "with tools which Tizen Studio provide."?

Also, we provide two extensions for Visual Studio and Visual Studio Code. In "[Visual Studio Tools for Tizen](/application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not only Tizen Web and Native applications but also Tizen .NET applications easily and rapidly _with Emulator, Certificate Manager, Device Manager, and Dynamic Analyzer_.

Also, we provide two extensions for Visual Studio and Visual Studio Code. In "[Visual Studio Tools for Tizen](/application/vstools/)" and "[Visual Studio Code extension for Tizen](/application/vscode-ext/)", you can develop not only Tizen Web and Native applications but also Tizen .NET applications easily and rapidly _with the useful developer tools_.
